astroquery/alma/core.py

```diff
@@ -197,44 +194,92 @@ def __init__(self):
         self._sia = None
         self._tap = None
         self._datalink = None
-        self.sia_url = None
-        self.tap_url = None
-        self.datalink_url = None
+        self._sia_url = None
+        self._tap_url = None
+        self._datalink_url = None
+
+    def service_id_auth(self, archive_host):
+        """
+        Convenience property to get the current Service ID authority.  Each ARC will have a different authority in
```

Comment:
It would be helpful to have a link to the appropriate documentation here if this is meant to be a public function.  I have no idea what a "Service ID authority" is, so I suspect users will not in general either.